.github/workflows/continuous-monitoring.yml

```diff
@@ -0,0 +1,19 @@
+name: Continuous monitoring of distribution channels
+on:
+  workflow_dispatch:
```

Comment:
Do we need `workflow_dispatch` here? I think this should be fine as well.

```
name: Do things every 6 hours
on:
  schedule:
    - cron: "* */6 * * *"
```

Reply:
I think it's useful to be about to manually run any cron job from ui or you always have to wait for the timer.